borsh-rs/fuzz/fuzz-run/src/main.rs

```diff
@@ -0,0 +1,102 @@
+use borsh::BorshDeserialize;
+use std::collections::{BTreeMap, HashMap, HashSet};
+
+#[macro_use]
+extern crate honggfuzz;
+
+macro_rules! fuzz_types {
+	(
```

Comment:
I suggest on each fuzz iteration we try deserializing all types, instead of just one. This will also make the macro much simpler since we wouldn't need `counter`, which means it wouldn't need a tt-muncher. tt-munchers are hard to read  IMO.

Reply:
done!